README.md

```diff
@@ -1,2 +1,75 @@
 # linux-workstation-data-engineering
-Collection of Ansible roles related to Data Engineering technologies to be run against UKHO/linux-workstation
+
+This is to be run on-top of the linux-workstation provided through `UKHO/linux-workstation`
+
+To run the playbook, run the following in the base of the repository:
+
+```
+$ sudo ansible-playbook ansible/playbook.yml -i ansible/inventories/laptop-vm/hosts --extra-vars="user_home=$HOME"
+$ cat ~/.bash_profile # You should see the following near the end of the file:
+...
+export PYENV_ROOT=/home/<your username>/.pyenv
+export PATH=/home/<your username>/.pyenv/bin:$PATH
+if command -v pyenv 1>/dev/null 2>&1; then eval "$(pyenv init -)";fi
+```
+
+## Ansible Roles
+
+* Docker
+* Docker Compose
+* Pyenv
+
+## Docker
+
+You may need to add yourself to the Docker group:
+
+```
+$ sudo usermod -aG docker <your username> # Log out and log back in
+$ docker ps # Check you can run a docker command
+CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
```

Comment:
Add the default user to the docker group via ansible as well as root - then this section of the README can be removed.

Reply:
Done